src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -22,7 +22,7 @@ public class ListCommandTest {
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(getTypicalInternBook(), new UserPrefs());
         expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
```

Comment:
i think you could maybe change the instances of addressbook in the ModelManager class to fit our context